cluster/common.sh

```diff
@@ -942,9 +942,9 @@ EOF
 DISABLE_NETWORK_SERVICE_SUPPORT: $(yaml-quote ${DISABLE_NETWORK_SERVICE_SUPPORT})
 EOF
     fi
-    if [ -n "${DISABLE_ADMISSION_PLUGINS:-}" ]; then
+    if [ -n "${NETWORK_PROVIDER:-}" ]; then
       cat >>$file <<EOF
-DISABLE_ADMISSION_PLUGINS: $(yaml-quote ${DISABLE_ADMISSION_PLUGINS})
+NETWORK_PROVIDER: $(yaml-quote ${NETWORK_PROVIDER})
```

Comment:
NETWORK_PROVIDER seems already have been processed that way.

Reply:
Yes, will remove this duplicate code.

Reply:
Done.